ash/src/vk/extensions.rs

```diff
@@ -755,7 +755,6 @@ impl ColorSpaceKHR {
     pub const BT709_NONLINEAR_EXT: Self = Self(1_000_104_006);
     pub const BT2020_LINEAR_EXT: Self = Self(1_000_104_007);
     pub const HDR10_ST2084_EXT: Self = Self(1_000_104_008);
-    pub const DOLBYVISION_EXT: Self = Self(1_000_104_009);
```

Comment:
This was deprecated, but perhaps we should keep it if it isn't deprecated because of "invalid name"?

Reply:
I've refactored our `deprecated` handling a bit to _only_ skip `"aliased"` reasons, which is where we obviously cannot generate proper names. All the rest gets a `#[deprecated]` attribute.